lib/rules/media-feature-name-no-vendor-prefix/index.js

```diff
@@ -34,6 +34,12 @@ function rule(actual) {
 				return;
 			}
 
+			if (context.fix) {
+				atRule.params = isAutoprefixable.unprefix(atRule.params);
```

Comment:
It's not enough, tests fail. And it's not clear for me. Maybe someone can help me to understand?

Reply:
fixed